src/main/java/at/ac/tuwien/damap/rest/persons/MockUniversityPersonServiceImpl.java

```diff
@@ -1,25 +1,35 @@
 package at.ac.tuwien.damap.rest.persons;
 
-import at.ac.tuwien.damap.rest.dmp.domain.ContributorDO;
-import io.quarkus.arc.DefaultBean;
-import org.eclipse.microprofile.rest.client.inject.RestClient;
+import java.util.List;
 
 import javax.enterprise.context.ApplicationScoped;
-import javax.inject.Inject;
-import java.util.List;
+
+import org.eclipse.microprofile.rest.client.inject.RestClient;
+
+import at.ac.tuwien.damap.rest.dmp.domain.ContributorDO;
+import io.quarkus.arc.DefaultBean;
+import lombok.extern.jbosslog.JBossLog;
 
 /*
     extend this class in your custom project, for your implementation
  */
 
 @ApplicationScoped
 @DefaultBean
```

Comment:
now that we have several services alongside each other, we should revise our approach with default beans and overwriting them.

Reply:
done